common.js

```diff
@@ -2716,6 +2716,44 @@ const allSoloCars = {
         '2004': ['ep'],
     },
   },
+  'Scion': {
+    'FR-S (incl. Release Series 1.0 and 2.0)': {
```

Comment:
This can be merged with the TRD and non-TRD FR-S models below. So basically, just add `stx` and `csp` to all of their class arrays.